votor/src/event_handler.rs

```diff
@@ -283,11 +283,11 @@ impl EventHandler {
                     received_shred,
                     stats,
                 )?;
-                if let Some((ready_slot, parent_block)) =
+                if let Some(parent_block) =
```

Comment:
Thanks for doing this. But for refactoring/renaming work like this, can we start to do them in the Agave repo so I don't need to keep syncing upstream?

We plan to fully move to agave in a few weeks (hopefully) and then we can only sync one way from Agave to Alpenglow.